tasks/base/config/appname.go

```diff
@@ -131,12 +143,13 @@ func (t BaseConfigAppName) Execute(options tasks.Options, upstream map[string]ta
 
 	return tasks.Result{
 		Status:  tasks.Success,
-		Summary: fmt.Sprintf("%s unique application name(s) found.", strconv.Itoa(len(appNameInfosFromConfig))),
+		Summary: fmt.Sprintf("%s unique application name(s) found: %s", strconv.Itoa(len(appNameInfosFromConfig)), appNameInfosFromConfig[0].Name),
```

Comment:
Just optimization but, since len() returns an int ([reference](https://golang.org/pkg/builtin/#:~:text=The%20len%20built%2Din%20function,len(v)%20is%20zero.)) we should be able to bypass the string conversion and just use something like %d for the first value

Reply:
Good catch! I will apply the suggested change.